src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/ConfirmApprenticeshipSteps.cs

```diff
@@ -0,0 +1,62 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using SFA.DAS.ApprenticeCommitments.Apis.InnerApi;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests.Steps
+{
+    [Binding]
+    public class ConfirmApprenticeshipSteps
+    {
+        private readonly TestContext _context;
+        private object _command;
+        private Fixture _fixture;
+
+        public ConfirmApprenticeshipSteps(TestContext context)
+        {
+            _context = context;
+            _fixture = new Fixture();
+        }
+
+        [Given("a Training Provider confirmation")]
+        public void GivenATrainingProviderConfirmation()
+        {
+            _command = new TrainingProviderConfirmationRequestData
+            {
+                TrainingProviderCorrect = true
+            };
+        }
+
+        [Given("the inner API will accept the confirmation")]
+        public void GivenTheInnerAPIWillAcceptTheConfirmation()
+        {
+            _context.InnerApi.MockServer
+                .Given(
+                    Request.Create()
+                        .WithPath("/apprentices/*/apprenticeships/*/trainingproviderconfirmation")
```

Comment:
My  concerns about the use of these wildcard matches are

1. Does it slow down the response from WireMock (I seem to see this but that anecdotal)
2. If we happen to pass in the wrong id in (or in the wrong position), our test will pass, but the code will fail

   Benefits are it's nicer than having to write explicit path for each case!